apps/modernization-api/src/main/java/gov/cdc/nbs/event/search/labreport/LabTestSummaryFinder.java

```diff
@@ -0,0 +1,64 @@
+package gov.cdc.nbs.event.search.labreport;
+
+import gov.cdc.nbs.patient.documentsrequiringreview.detail.LabTestSummary;
+import gov.cdc.nbs.patient.documentsrequiringreview.detail.LabTestSummaryRowMapper;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.stereotype.Component;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class LabTestSummaryFinder {
+  private static final String QUERY = """
+      select
+        lt.lab_test_desc_txt    as name,
+        cvg.code_short_desc_txt as status,
+        lr.lab_result_desc_txt  as coded_result,
+        ovn.numeric_value_1     as numeric_result,
+        ovn.high_range          as high,
+        ovn.low_range           as low,
+        ovn.numeric_unit_cd     as unit
+      from
+        NBS_SRTE..Lab_test lt
+        join observation o on lt.lab_test_cd = o.cd and o.obs_domain_cd_st_1 = 'Result'
```

Comment:
I think this is missing a join, `Act_relationship`  links a Lab Report (Observation) to the Lab Test Results (also Observation) with the Lab Report being the `target_act_uid`